pkg/registry/servicecatalog/broker/storage.go

```diff
@@ -143,5 +145,27 @@ func NewStorage(opts server.Options) (brokers, brokersStatus rest.Storage) {
 	statusStore := store
 	statusStore.UpdateStrategy = brokerStatusUpdateStrategy
 
-	return &store, &statusStore
+	return &store, &StatusREST{&statusStore}
+}
+
+// StatusREST defines the REST operations for the status subresource.
```

Comment:
Perhaps add:
```
This resource is a, sort of, proxy for the real ServiceBroker resource,
however, by only defining the Get and Update functions it will limit
which HTTP verbs are allowed to operate over it.
```
Assuming I understand what's going on.